docs/DeveloperGuide.md

```diff
@@ -286,78 +288,158 @@ Priorities: High - (must have), Medium - (nice to have), Low -  (unlikely to hav
   </tr>
   <tr>
     <td>HIGH</td>
-    <td>Keep track of my client's phone number</td>
-    <td>I can call them to renew their gym membership</td>
+    <td>View all ingredients in my inventory</td>
+    <td>I can see the remaining quantity of all my ingredients</td>
   </tr>
   <tr>
     <td>HIGH</td>
-    <td>Add memberships to clients by period</td>
-    <td>I do not have to manually key in the expiry date</td>
+    <td>Detect duplicate ingredients</td>
+    <td>I do not clutter the application with duplicate ingredients</td>
   </tr>
   <tr>
     <td>HIGH</td>
-    <td>Search for a order by a client's name</td>
-    <td>I can confirm if they have a order before letting them enter</td>
+    <td>Add a client</td>
+    <td>I can copy details from a client to easily fill up the details of an order</td>
   </tr>
   <tr>
     <td>HIGH</td>
     <td>Delete clients</td>
-    <td>I will not clutter my app</td>
+    <td>I can remove former clients</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Edit client details</td>
+    <td>I can update their contact information</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Find clients by keywords</td>
+    <td>I can find client information to fill up the details of an order</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>View all clients</td>
+    <td>I can view all my current clients</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Detect duplicate clients</td>
+    <td>I do not clutter the application with duplicate clients</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Add an order</td>
+    <td>I can keep track of my orders</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Add a quantity to an order</td>
+    <td>I can create multiple orders for the same client</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Add a price to an order</td>
+    <td>I can keep track of my revenue</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Delete orders</td>
+    <td>I can delete cancelled orders</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Edit orders</td>
+    <td>I can update the details of my orders</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Find orders by keywords</td>
+    <td>I can find a specific order</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>View all orders</td>
+    <td>I can view all my current orders</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Mark an order as done</td>
+    <td>I can set the status of an order</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Mark an order as undone</td>
+    <td>I can reverse the changes to made to the status of an order</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Add a recipe</td>
+    <td>I can copy details from a recipe to easily fill up the details of an order</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Delete recipes</td>
+    <td>I can delete unused recipes</td>
+  </tr>
+  <tr>
+    <td>HIGH</td>
+    <td>Edit recipes</td>
+    <td>I can update the ingredients used in recipes</td>
```

Comment:
Maybe can change to this if we're allowing them to make changes to more than just the ingredients
```suggestion
    <td>I can update recipe details, like changing the ingredients used or recipe name</td>
```